crates/autopilot/src/run_loop.rs

```diff
@@ -54,6 +55,7 @@ pub struct RunLoop {
     pub score_cap: U256,
     pub max_settlement_transaction_wait: Duration,
     pub solve_deadline: Duration,
+    pub in_flight_orders: Arc<Mutex<InFlightOrders>>,
```

Comment:
Why do we need synchronisation here?

Reply:
Because we populate the data in a method that takes a shared reference to `self`.